src/components/richLink.tsx

```diff
@@ -136,6 +136,10 @@ const richLinkStyles = (format: Format): SerializedStyles => {
 		margin: 0.375rem ${remSpace[4]} ${remSpace[3]} 0;
 
 		width: ${richLinkWidth};
+		@media (max-width: 23.4rem) {
+			width: 100%;
+			box-sizing: border-box;
```

Comment:
We will also need to hide the rich link image here, so it doesn't look like it's part of the story.